package lk.ijse.challengetest.exeption;

public class UserNotFoundExeption extends RuntimeException{
    public UserNotFoundExeption() {
    }
    public UserNotFoundExeption(String message) {
    }
    public UserNotFoundExeption(String message, Throwable cause) {
    }
}
